torch/ao/sparsity/_experimental/data_sparsifier/base_data_sparsifier.py

```diff
@@ -74,8 +78,15 @@ def _extract_weight(self, data):
         elif type(data) in EMBEDDING_TYPES:
             return data.weight
 
-    def add_data(self, name: str, data, **config):
-        r""" Configures and parametrizes the internal container model with name and data
+    def add_data(self, name: str, data, use_old_mask=True, **config):
```

Comment:
Can we rename the `use_old_mask` into `reuse_mask` (or something like that)? imho, it's avoids the questions, such as "what do you mean by old?"